src/styles/theme/defaultTheme.ts

```diff
@@ -336,5 +375,61 @@ export const defaultTheme = createTheme({
         noSsr: true,
       },
     },
+    MuiInputBase: {
+      styleOverrides: {
+        root: {
+          padding: 0,
+        },
+        input: {
+          padding: '0 !important',
+          backgroundColor: COLORS.white,
+          color: COLORS.grayDark,
+          '&::placeholder': {
+            color: 'rgba(49,67,90,0.7)',
+          },
+        },
+        adornedStart: ({ theme }) => ({
+          padding: theme.spacing(4),
+        }),
+      },
+    },
+    MuiOutlinedInput: {
+      styleOverrides: {
+        root: {
+          padding: 0,
+        },
+        notchedOutline: {
+          border: 'none',
+        },
+      },
+    },
+    MuiTextField: {
```

Comment:
Search input is highly customised and I am not sure adjusting "generic" inputs will work long term, but we will see.

Reply:
Right, I just put it into theme, so it's easier to refactor in the future. But agree that it is not good long term solution.